benefits/static/css/styles.css

```diff
@@ -52,6 +52,10 @@
   /* prettier-ignore */
 }
 
+html {
+  scroll-padding-top: 120px;
```

Comment:
Can you please add a little comment here on why this is necessary / the significance of the value?

Reply:
I honestly just tried out a bunch of numbers until it worked.

Reply:
Figured it out! 

https://github.com/cal-itp/benefits/pull/1663/commits/3711239b692f18f0ed4ad749ed55f0dfa2fbba47

Interesting observation: In Chrome, it works with a padding-top of 80px, but doesn't for Firefox. I had to test on both browsers, but ultimately Firefox needed a larger number. Ended up with:

```css
scroll-padding-top: 101px; /* Header Height (80px) + Skip Nav Height (21px) = 101px */
```